### ISSUE TITLE
image code broken

### ISSUE THREAD

**Issue by fyears** [readthedocs/recommonmark#88]
The pypi version works fine.

However, fter installing the latest recommonmark [this commit](https://github.com/rtfd/recommonmark/tree/c410abb565a3c4ad4de0ce204ab2fa2db8031cf1) on GitHub by:

```
pip install git+https://github.com/rtfd/recommonmark
```

The image code gets broken:

```markdown
![alt text](images/example.png)
```

is translated into

```html
<p><img alt="_images/example.png" src="_images/example.png" />alt text</p>
```

instead of the correct one:

```html
<img alt="alt text" src="_images/example.png" />
```

**Comment by johnpemberton** [comment]
Digging into the source code [here](https://github.com/rtfd/recommonmark/blob/589d25994a86695e5fd13ba8695bd015c76ce87f/tests/test_sphinx.py#L129) and [here](https://raw.githubusercontent.com/rtfd/recommonmark/master/tests/sphinx_generic/index.md), I figured out that images work in the following way: 
```markdown
![caption](images/example.png "alt-text")
```
results in:
```html
<p><img alt="alt-text" src="_images/example.png" />caption</p>
```
The current workaround I'm using is to leave the caption empty:

```markdown
![](images/example.png "alt-text")
```
resulting in:
```html
<p><img alt="alt-text" src="_images/example.png" /></p>
```

**Comment by nsoranzo** [comment]
So basically alt text and image title are swapped with respect to the official Markdown specs: https://daringfireball.net/projects/markdown/syntax#img

**Comment by chris-c-thomas** [comment]
@nsoranzo no I think it's more than that. When I build a markdown file it takes the alt text and places it outside the image tag but inside a paragraph tag.

In doing so, the alt text displays next to the image as opposed to inside the alt text attribute of the img  tag.

**Comment by chris-c-thomas** [comment]
**Example Markdown**

`![alt text](images/example.png)`

becomes:

`<p><img alt="images/example.png" src="images/example.png">alt text</p>`

So the path is being placed into both the `alt` and `src` attributes and the alt text is being placed as content inside the `<p></p>` tags.

**Comment by nsoranzo** [comment]
This can be closed, fixed in #150.
